test/Graphics/Drawable.test.cpp

```diff
@@ -7,3 +10,40 @@ static_assert(!std::is_copy_constructible_v<sf::Drawable>);
 static_assert(std::is_copy_assignable_v<sf::Drawable>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Drawable>);
 static_assert(std::is_nothrow_move_assignable_v<sf::Drawable>);
+
+class DrawableTest : public sf::Drawable
+{
+public:
+    void draw(sf::RenderTarget&, const sf::RenderStates&) const override
+    {
+        m_called += 1;
+    }
+    int callCount() const
+    {
+        return m_called;
+    }
+
+protected:
+    mutable int m_called{};
+};
+
+TEST_CASE("[Graphics] sf::Drawable")
+{
+    SUBCASE("Construction")
+    {
+        const DrawableTest drawableTest;
+        REQUIRE(drawableTest.callCount() == 0);
+    }
+
+    SUBCASE("draw()")
+    {
+        const DrawableTest drawableTest;
+        sf::RenderTexture  renderTexture;
+        if (renderTexture.create({32, 32}))
+        {
+            REQUIRE(drawableTest.callCount() == 0);
+            renderTexture.draw(drawableTest);
+            REQUIRE(drawableTest.callCount() == 1);
+        }
```

Comment:
I think that `if` should become another test assertion so that if it fails, we don't just quietly stop running the tests contained within that block.